storage/disk/disk.go

```diff
@@ -506,6 +488,15 @@ func (db *Store) Commit(ctx context.Context, txn storage.Transaction) error {
 		for h := range db.triggers {
 			h.cb(ctx, readTxn, event)
 		}
+
+		// cleanup backup db
+		if db.backupDB != nil {
+			if err := db.cleanup(db.backupDB); err != nil {
```

Comment:
[nit] Since we're already calling `db.cleanup`, why pass `db.backupDB`? It's available to the function body anyways.

[update] Ah, below we're passing something else. 💡